src/cdk/drag-drop/drop.ts

```diff
@@ -30,6 +30,12 @@ import {moveItemInArray} from './drag-utils';
 /** Counter used to generate unique ids for drop zones. */
 let _uniqueIdCounter = 0;
 
+/**
+ * Buffer, in percent of the width/height, to add around the drop container when
+ * determining whether an item should affect the list order.
+ */
+const DROP_BUFFER_PERCENTAGE = 5;
```

Comment:
I think that eventually we should allow people to customize the value, rather than picking a min and max since this is very much down to personal preference. The 5% value should do for now, because it's small enough that even if a drop container takes up the entire viewport on a 1080p screen, it would still only be 96px.